manifests/cf-manifest/scripts/generate-cf-certs.sh

```diff
@@ -18,6 +18,9 @@ router_ssl,${ROUTER_DOMAINS}
 uaa_jwt_signing,
 consul_server,server.dc1.cf.internal,server.dc2.cf.internal
 consul_agent,
+logsearch,logsearch.${SYSTEM_DNS_ZONE_NAME}
+metrics,metrics.${SYSTEM_DNS_ZONE_NAME}
+router,${ROUTER_DOMAINS}
```

Comment:
As discussed in person, its ok to have both, as we want to replace the external facing elb certficate in production environments, but not use this on the gorouter. But we should rename them.
However, the current logic [requires the certificate to be called `router_ssl`](https://github.com/alphagov/paas-cf/blob/move_public_facing_tls_termination_to_elbs_115709073/concourse/pipelines/create-bosh-cloudfoundry.yml#L811-L812) because that matches the required field in the manifest - we could special case it, but i'm not sure its worth it...